tensorflow/python/keras/utils/losses_utils_test.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for losses_utils."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import test_util
+from tensorflow.python.framework import ops
+from tensorflow.python.keras import combinations
+from tensorflow.python.keras.utils import losses_utils
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops.ragged import ragged_array_ops
+from tensorflow.python.ops.ragged import ragged_concat_ops
+from tensorflow.python.ops.ragged import ragged_factory_ops
+from tensorflow.python.platform import test
+
+
+@combinations.generate(combinations.combine(mode=['graph', 'eager']))
+class RemoveSqueezableTest(test_util.TensorFlowTestCase):
+  """Test remove_squeezable_dimensions
+  """
+  def test_ragged_3d_same_shape(self):
+    """ shape (2, (sequence={1, 2}), 3)
+    """
+    x = ragged_concat_ops.stack([
```

Comment:
[Optional] There's no need to use stack here -- you could just use:
```
x = ragged_factory_ops.constant([[[1, 2, 3]], [[4, 5, 6], [7, 8, 9]])
```

If you want the innermost dimension to have a dense size of 3 (rather than a ragged size, where each row happens to have length 3), you could instead use:
```
x = ragged_factory_ops.constant([[[1, 2, 3]], [[4, 5, 6], [7, 8, 9]], ragged_rank=1)
```

Reply:
Done.